ports/raspberrypi/boards/lilygo_t_display_rp2040/pins.c

```diff
@@ -10,9 +10,11 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_GPIO2) },
     { MP_ROM_QSTR(MP_QSTR_GP2), MP_ROM_PTR(&pin_GPIO2) },
 
+    { MP_ROM_QSTR(MP_QSTR_BUTTON0), MP_ROM_PTR(&pin_GPIO6) },
     { MP_ROM_QSTR(MP_QSTR_BUTTON_L), MP_ROM_PTR(&pin_GPIO6) },
```

Comment:
I didn't want to break anyone who is already using it that way.

Reply:
I wouldn't mind but had the same thought as Scott - since its already out there it might break the code of some users. We can have it clean for the T-PicoC3 (same just with Wifi added) and have only BUTTON0 and BUTTON1. I see already more direct offers for the T-PicoC3 than the T-Display rp2040.